trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0...10]
```

Comment:
- [ ] Kindly consider the name has a maximum of 10 characters. If it's longer it should trim the word.
```ruby
@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```